examples/end-to-end-session-based/03-End-to-end-session-based-with-Yoochoose-TF.ipynb

```diff
@@ -0,0 +1,1015 @@
+{
```

Comment:
<p>As currently <code>mlm</code> is overfitting, I guess we should mention <code>clm</code>    I think we could remove the reference to taskinstead of <code>mlm</code> in the second paragraph.  Also we could maybe remove the reference to <code>tutorial</code> folder for the moment as we don't support a <code>Tf</code> version ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Sure.